CHANGELOG.md

```diff
@@ -1,5 +1,52 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2025-02-20
+
+Monitor the release status by region at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled `v20250220`.
+
+### Announcements
+* 
+
+### Release Notes
+* Features:
+   * AKS Kubernetes patch versions 1.29.12, 1.29.13, 1.30.8, 1.30.9, 1.31.4, and 1.31.5 are now available.
+   * 
+
+
+* Preview Features:
+   * AKS Kubernetes version 1.32 is now in preview.
+   * [Kubelet serving certificate rotation](https://learn.microsoft.com/Azure/AKS/certificate-rotation) now allows AKS to utilize kubelet server TLS bootstrapping for both bootstrapping and rotating serving certificates signed by the Cluster CA.
```

Comment:
```suggestion
```
This is only available in non-public regions. Not to be included in release notes until public